src/services/mongoose/users.ts

```diff
@@ -88,6 +86,36 @@ export const UsersRepository = (): IUsersRepository => {
     }
   }
 
+  const migrateUserIdSubject = async ({
+    currentUserIdSubject,
+    newUserIdSubject,
+    phone,
+  }: {
+    currentUserIdSubject: UserId
+    newUserIdSubject: UserId
+    phone: PhoneNumber
+  }): Promise<User | RepositoryError> => {
+    const filter = { userId: { $eq: currentUserIdSubject } }
```

Comment:
is not the same to use `{ userId: currentUserIdSubject }`

Reply:
it fails the codeql code scan without the `$eq`